apps/lnbits/docker-compose.yml

```diff
@@ -34,7 +34,7 @@ services:
       # App
       LNBITS_SITE_TITLE: "LNbits - Umbrel"
       LNBITS_DEFAULT_WALLET_NAME: "LNbits wallet"
-      LNBITS_DISABLED_EXTENSIONS: "amilk"
+      LNBITS_DISABLED_EXTENSIONS: "amilk,lndhub"
```

Comment:
I think you shouldn't disable LNDHub here. Because the whole point of LNbits is offer the ability to have multiple users on one node. Each wallet / user has its own LNDHub and thus can connect their BlueWallet to it, while the LNDHub app is for only one user (= the node wallet). 

At least it is what I've understood, correct me if I'm wrong.

Reply:
its exactly the opposite. lnbits just replicated lndhub. poorly.

Reply:
Yeah, LndHub offers multiple accounts too, and each LNbits user having multiple sub-acoounts would be a bit to complicated.

Reply:
@Overtorment are there known issues with LNbits LNDhub implementation?

I think there is a slightly different use case between BlueWallet Server/LNbits+LNDhub here.

You can create sub accounts in LNbits for things like earnings from a paywall link forwarder. Then you can add that LN wallet to BlueWallet to track your earnings from the paywall. You could also then easily sweep funds to your main wallet via BlueWallet without having to access the LNbits web wallet etc.

I see it more as a way to just interface with the LNbits wallets via BlueWallet rather than a serious LNDhub replacement.

So:
- LNbits+LNDhub: Access your LNbits wallets from BlueWallet.
- BlueWallet Server: Back your BlueWallet with your own Lightning node.

What do you think?

Reply:
Thing here is "lndhub" should never be visible to the user, thus the confusion, there's nothing people can do there. Should just be a kid of "BlueWallet Server" as a way to connect to the app. If lnbits/lndhub connection won't break lndhub, all is good.